Meadow_DotNet_SDK/Project_Templates/templates/Meadow.StartKit/StartKit.Core/Commands/ChangeSetpointsCommand.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Meadow.Cloud;
+
+namespace $safeprojectname$.Core
```

Comment:
@jorgedevs This is 3. Note how namespaces and uses and even classes or interfaces are named with the substitute variable.